cloudformation/s3_config.yaml

```
@@ -0,0 +1,22 @@
+  AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  InsecureS3Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: coderabbit-s3-demo  
+      AccessControl: PublicRead  
```

yamllint

[error] trailing spaces

(trailing-spaces)